rigs/component/profiles/react/config/jest.config.json

```diff
@@ -24,5 +24,8 @@
       "lines": 100,
       "statements": 100
     }
-  }
+  },
+
+  "silent": true,
+  "verbose": true
```

Comment:
Having both of these set to true seems funny 🤷

Reply:
i know it does, they aren't opposites though